lib/zuora/objects/rate_plan_charge.rb

```diff
@@ -54,12 +54,12 @@ class RatePlanCharge < Base
     validates_inclusion_of    :overage_unused_units_credit_option, :allow_nil => true, :in => %w(NoCredit CreditBySpecificRate), :if => Proc.new { |rpc| ['Overage','Tiered with Overage Pricing'].include?(rpc.charge_model) }
     validates_numericality_of :price, :allow_nil => true
     validates_numericality_of :price_increase_percentage, :less_than_or_equal_to => 100, :greater_than_or_equal_to => -100, :allow_nil => true
-    validates_datetime_of     :processed_through_date, :allow_nil => true
+    # validates_datetime_of     :processed_through_date, :allow_nil => true
     validates_numericality_of :quantity, :allow_nil => true, :greater_than_or_equal_to => 0
     validates_numericality_of :rollover_balance, :allow_nil => true
     validates_numericality_of :segment, :integer_only => true, :greater_than_or_equal_to => 1
     validates_numericality_of :tcv
-    validates_datetime_of     :trigger_date, :allow_nil => true
+    # validates_datetime_of     :trigger_date, :allow_nil => true
```

Comment:
More commented out validations, which weren't commented out previously. Do we know why?

Reply:
Resolved here [04f281f](https://github.com/Sage/zuora/pull/18/commits/04f281f9a0a0b07961e1b1f4ebd5465beb8c27bc)